contracts/IdleYieldSource.sol

```diff
@@ -76,6 +74,17 @@ contract IdleYieldSource is IProtocolYieldSource, Initializable, ReentrancyGuard
         emit IdleYieldSourceInitialized(address(_idleToken));
     }
 
+    /// @notice Approve Idle token contract to spend max uint256 amount
+    /// @dev Emergency function to re-approve max amount if approval amount dropped too low
+    /// @return true if operation is successful
+    function approveMaxAmount() external returns (bool) {
```

Comment:
Since we have an `owner` for this contract, let's lock this function down in the event of an exploit through idle.

Reply:
Done: https://github.com/pooltogether/idle-yield-source/pull/5/commits/a2716460b5e3d695562dd1cb05e211263fbf7c69